lab-dustin/lib/storage.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const createError = require('http-errors');
+
+const writeFileAsync = promisify(fs.writeFile);
+const readFileAsync = promisify(fs.readFile);
+const deleteFileAsync = promisify(fs.unlink);
+
+module.exports = exports = {};
+
+exports.createItem = function(schemaName, item) {
+  if (!schemaName) return Promise.reject(new Error('expected schemaName'));
+  if (!item) return Promise.reject(new Error('expected item'));
```

Comment:
These (and similar validation errors below) should use `http-errors` to specify how to reject this as a bad request.